LibOpenRaid.lua

```diff
@@ -3195,6 +3210,284 @@ openRaidLib.commHandler.RegisterComm(CONST_COMM_COOLDOWNREQUEST_PREFIX, openRaid
     openRaidLib.internalCallback.RegisterCallback("onEnterGroup", openRaidLib.KeystoneInfoManager.OnPlayerEnterGroup)
     openRaidLib.internalCallback.RegisterCallback("mythicDungeonEnd", openRaidLib.KeystoneInfoManager.OnMythicDungeonFinished)
 
+--------------------------------------------------------------------------------------------------------------------------------
```

Comment:
I essentially copied and modified the keystones code, so the structure should be identical